### ISSUE TITLE
Feedback - Links

### ISSUE THREAD

**Issue by daraghbroderick** [raspberrypilearning/cd-beginner-app-inventor-sushi#7]
"Hi Team. For the projects tagged 'App Inventor' many of them don't include instructions or links to the MIT App Inventor website, which they would need to access in order to complete the project: https://projects.raspberrypi.org/en/projects?software[]=app-inventor.
I'd suggest including text such as this in the Introduction section: "App Inventor is a cloud-based tool, which means you can build apps right in your web browser. Visit the MIT App Inventor website (https://appinventor.mit.edu/) to explore their support resources and guides. When you’re ready to start building your own app, visit ai2.appinventor.mit.edu or click the ‘Create Apps!’ button on the MIT App Inventor homepage."
Related note: there's a lack of context regarding the sentences like "These are the advanced App Inventor Sushi Cards" and "You may need a mentor at your Dojo to help you with this step!" Since these projects are accesible to anyone, these statements will be confusing to those not familiar with or involved in CoderDojo, and so I'd recommend either removing or adding a sentence to explain further and provide context."

**Comment by daraghbroderick** [comment]
Same applies to all at https://projects.raspberrypi.org/en/projects?software%5B%5D=app-inventor

**Comment by libxx1** [comment]
@PhilipHarney hiya, do you think you could come up with a sentence or two for the App Inventor introduction page along the lines of Daragh's suggestion please. If you post it as a comment here I can update the App Inventor projects. 

I'll go through the projects and take out references to dojo/mentors so they can be used by anyone.

**Comment by libxx1** [comment]
A user brought this up again today via project feedback on Accessibility Map.

**Comment by libxx1** [comment]
A user brought this up again, this time in Exercise Tracker "Hi - just glancing through this project.  There is no link to App Inventor or directions for getting setup.   It looks like the startup directions are in Global Food.  I'd suggest some reference/crosslink/prereq."

**Comment by PhilipHarney** [comment]
Yeah… we can do the small fix of adding directions to the app inventor site, but I'd see these as a good candidate for retirement, eventually, unless we plan to double-down on App Inventor as a supported language/tool.

**Comment by libxx1** [comment]
Hey @PhilipHarney , a couple of feedback items came in on the Global Food App project. Just popping them here in case you want to cover them in your updates to the app inventor projects. 

URL: /en/projects/cd-sebento-appinv-1/4

When you open another screen it is advised to close the previous one. Else you could get memory problems on the phone.

HI! Great resource!! BTW, I found a small typo on https://projects.raspberrypi.org/en/projects/cd-sebento-appinv-1/4  - near bottom of page. Pls change: "Of course, that only works if Button1 is clicked." to refer to Button4 .  Thank you!!
